rundeck/resource_job.go

```diff
@@ -321,13 +321,37 @@ func resourceRundeckJob() *schema.Resource {
 							Optional: true,
 							Elem:     resourceRundeckJobPluginResource(),
 						},
+						"error_handler": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem:     resourceRundeckJobCommandResource(),
+						},
 					},
 				},
 			},
 		},
 	}
 }
 
+func resourceRundeckJobCommandResource() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"keep_going_on_success": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"config": {
```

Comment:
This should probably be a typed schema. It matches very closely to the command schema, though the differences may make it tricky to compose in a DRY fashion via the SDK and go..